ru/news/_posts/2018-03-28-ruby-2-5-1-released.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: news_post
+title: "Вышел Ruby 2.5.1"
+author: "naruse"
+translator: "arbox"
+date: 2018-03-28 17:20:00 +0000
+lang: ru
+---
+
+Вышел Ruby 2.5.1.
+
+Этот релиз влючает в себя исправление ошибок и решений проблем в безопасности.
+
+* [CVE-2017-17742: HTTP response splitting in WEBrick](/en/news/2018/03/28/http-response-splitting-in-webrick-cve-2017-17742/)
```

Comment:
@arbox let's translate descriptions?

Reply:
@razum2um we definitely need to...
I'd like to translate all related security releases and update links to them.